### ISSUE TITLE
Application crashes when run on a platform without a system tray

### ISSUE THREAD

**Issue by marianpetras** [ycnmhd/ucvk#1]
The application crashes during startup on a platform without a system tray, e.g. Ubuntu 18.04.1.
Brief exception stack-trace:
```
java.lang.reflect.InvocationTargetException
	...
Caused by: java.lang.RuntimeException: Exception in Application init method
	...
Caused by: java.lang.UnsupportedOperationException: The system tray is not supported on the current platform.
	at java.desktop/java.awt.SystemTray.getSystemTray(Unknown Source)
	at org.dots_and_hyphens.main.java.core.fx$toolbar$tray_icon.<init>(fx.java:2220)
	at org.dots_and_hyphens.main.java.core.fx$toolbar.<init>(fx.java:1952)
	at org.dots_and_hyphens.main.java.core.fx.<init>(fx.java:114)
	at org.dots_and_hyphens.main.java.core.ucvk.init(ucvk.java:84)
	... 3 more

**Comment by ycnmhd** [comment]
Hi. I cannot simulate your environment to reproduce the issue. could you please test this version and report if the error is fixed?
[ucvk.zip](https://github.com/ycnmhd/ucvk/files/2299063/ucvk.zip)

**Comment by marianpetras** [comment]
I confirm I can run the attached version.

**Comment by ycnmhd** [comment]
Good! The fix has been implemented in the new release  (0.1.7.5)
